brushfire-core/src/main/scala/com/stripe/brushfire/AnnotatedTree.scala

```diff
@@ -93,84 +122,84 @@ case class AnnotatedTree[K, V, T, A: Semigroup](root: Node[K, V, T, A]) {
   /**
    * Returns the leaf with index `leafIndex` by performing a DFS.
    */
-  def leafAt(leafIndex: Int): Option[LeafNode[K, V, T, A]] = leafAt(leafIndex, root)
+  def leafAt(leafIndex: Int): Option[LeafNode[K, V, T, A]] =
+    leafAt(leafIndex, root)
 
   /**
-   * Returns the leaf with index `leafIndex` that is a descendant of `start` by
-   * performing a DFS starting with `start`.
+   * Returns the leaf with index `leafIndex` that is a descendant of
+   * `start` by performing a DFS starting with `start`.
    */
-  def leafAt(leafIndex: Int, start: Node[K, V, T, A]): Option[LeafNode[K, V, T, A]] = {
+  def leafAt(leafIndex: Int, start: Node[K, V, T, A]): Option[LeafNode[K, V, T, A]] =
```

Comment:
There's only one use of this method in the trainer -- I'll see if I can replace it with a traversal.